### PR TITLE
feat: [IDP-1773]: cookiecutter step added

### PR DESCRIPTION
**Pull request by bhavya-sinha**
### Summary
**Will add coverage in follow up PR**

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [x] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

**Comment by bhavya-sinha** [comment]
trigger messagecheck

File: package.json [hunk 1]
```diff
@@ -62,8 +62,8 @@
     "@emotion/styled": "^10.0.27",
     "@harness/design-system": "1.5.2",
     "@harness/help-panel": "1.14.0",
-    "@harness/icons": "^1.211.0",
-    "@harness/ng-tooltip": "^1.31.204",
+    "@harness/icons": "1.212.0",
```
Comment (chetanvekariya-harness) [inline package.json:65]: Should we keep `^` as it is?

File: package.json [hunk 2]
```diff
@@ -62,8 +62,8 @@
...
+    "@harness/ng-tooltip": "^1.31.199",
```
Comment (chetanvekariya-harness) [inline package.json:66]: Is this intended ?

Comment (bhavya-sinha) [reply package.json:66]: ahhh no. happened while resolving conflicts. Will fix

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by jenil-harness — APPROVED** [review]
LGTM

**Pull request merged**